libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.scss

```diff
@@ -134,4 +134,23 @@ ion-content {
     --padding-end: #{utils.size('xxxl')};
   }
 }
+
+:host(.collapsible-title) {
+  ion-content {
+    --padding-top: 0px;
+
+    ion-header ion-toolbar:first-of-type {
+      padding-top: 0px;
```

Comment:
Note to self: We should merge main into v6, so we get prettier and stylelint update onto this. 
0px should be 0 here. 

0px is okay for custom props though!

Reply:
Should i do anything specific here 👀?

Reply:
Could you try to run `npm run prettier:fix` and `npm run lint:designsystem:css`?